src/ray/object_manager/ownership_based_object_directory.cc

```diff
@@ -186,6 +187,9 @@ void OwnershipBasedObjectDirectory::ReportObjectSpilled(
   update.mutable_spilled_location_update()->set_spilled_url(spilled_url);
   update.mutable_spilled_location_update()->set_spilled_to_local_storage(
       spilled_to_local_storage);
+  if (!generator_id.IsNil()) {
+    update.set_generator_id(generator_id.Binary());
```

Comment:
hmm looks it's not used?

Reply:
It gets used in the spill location handler.